ballerina/Dependencies.toml

```diff
@@ -28,7 +28,7 @@ modules = [
 [[package]]
 org = "ballerina"
 name = "io"
-version = "1.3.1"
```

Comment:
Need to update the version and jar names in here as well
https://github.com/ballerina-platform/module-ballerina-file/blob/master/ballerina/Ballerina.toml

Reply:
Updated in [ba252f9](https://github.com/ballerina-platform/module-ballerina-file/pull/475/commits/ba252f947b23fd76d69c8220f08fc165d4d62924)